### DISCUSSION TITLE
DietPi-Software | Mediawiki

### DISCUSSION THREAD

**Discussion by Ernstian** [MichaIng/DietPi#5589] [Software Request] [answered]
#### Formal software information
- Software title | Mediawiki
- Short description | A wiki system for organizing information
- Official URL (if available) | https://www.mediawiki.org/wiki/MediaWiki
- GitHub URL (if available) | https://github.com/wikimedia/mediawiki
- (Official) install documentation | https://www.mediawiki.org/wiki/Special:MyLanguage/Manual:Installation_guide

#### Are there similar/alternative software titles available with DietPi-Software?
- None (I tried the Nextcloud app "Collectives", but it's not as structured as I'd like it to be)

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
- There are many moving parts - PHP, Database, server, which all need the right version and (probably) not properly hardened

#### Can you provide the installation steps that you would suggest DietPi-Software to do?
1. Installing a fitting PHP version
2. Installing a fitting databse
3. Installing a fitting web server
4. Installing Mediawiki

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.
- I have no idea what I'm doing, sorry.

**Comment by Joulinar** [comment]
I moved the request to GitHub discussions. Vote for this software implementation with the ⬆️ button on this post.

**Reply by Ernstian** [reply]
Thank you! Sorry for posting it as an issue, I went via the [Contribute](https://dietpi.com/contribute.html) page on dietpi.com.

**Reply by Joulinar** [reply]
All ok 😂

**Reply by Joulinar** [reply]
PR up: https://github.com/MichaIng/DietPi/pull/6443

**Reply by Joulinar** [reply, marked as answer]
Added to DietPi V8.19
